clients/tests.py

```diff
@@ -1,3 +1,106 @@
-from django.test import TestCase
+from rest_framework.test import APITestCase
+from rest_framework import status
+from django.urls import reverse
+from .models import Client
+from django.contrib.auth import get_user_model
+from rest_framework_simplejwt.tokens import RefreshToken
 
-# Create your tests here.
+class ProductTests(APITestCase):
```

Comment:
If you are testing client features you should not name this test case as `ProductTests`.